omisego-admin/README.md

```diff
@@ -1 +1,264 @@
+
+Lastly, you will need to pass the instance that you got from the previous step to the `OMGAPIAdmin`'s constructor.
+
+For example,
+```kotlin
+ val config = AdminConfiguration(baseURL = "YOUR_BASE_URL")
+
+ val eWalletAdmin = EWalletAdmin.Builder {
+     clientConfiguration = config
+     debug = false
+ }.build()
+ 
+ val omgAPIAdmin = OMGAPIAdmin(eWalletAdmin)
+```
+
+Where:
+* `baseURL` is the URL of the OmiseGO Wallet API. **Note**: This need to be ended with '/'.
+* `debug` *(Optional)* is a boolean indicating if the SDK should print logs in the console. default: `false`. 
+
+## Retrieving resources
+
+Once you have an `OMGAPIAdmin` object from the [initialization section](#initialization), you can retrieve different resources.
+Every call takes an `OMGCallback` interface that returns an `OMGResponse` object:
+
+```kotlin
+interface OMGCallback<in T> {
+    fun success(response: OMGResponse<T>)
+    fun fail(response: OMGResponse<APIError>)
+}
+```
+
+```kotlin
+data class OMGResponse<out T>(val version: String, val success: Boolean, val data: T)
+
+data class APIError(val code: ErrorCode, val description: String)
+```
+
+Where:
+
+`success` is the function invoked when the `success` boolean in the response is `true`. This function will provide the corresponding data model to an API endpoint.
+
+`fail` is the function invoked when the `success` boolean in the response is `false`. This function will provide the `APIError` object which contains information about the failure.
+
+### Login
+
+```kotlin
+val params = LoginParams(
+    "john.doe@omise.co",
+    "password"
+)
+
+omgAPIAdmin.login(params).enqueue(object: OMGCallback<User>{
+    override fun fail(response: OMGResponse<APIError>) {
+        
+    }
+
+    override fun success(response: OMGResponse<AuthenticationToken>) {
+        
+    }
+})
+```
+
+> Note: The `Authentication` header will be set automatically when login successfully.
+
+### Transferring token
+
+```kotlin
+val request = TransactionCreateParams(
+    fromAddress = "1e3982f5-4a27-498d-a91b-7bb2e2a8d3d1",
+    toAddress = "2e3982f5-4a27-498d-a91b-7bb2e2a8d3d1",
+    amount = 1000.bd,
+    tokenId = "BTC:xe3982f5-4a27-498d-a91b-7bb2e2a8d3d1",
+    idempotencyToken = "some token"
+)
+
+omgAPIClient.transfer(request).enqueue(object : OMGCallback<Transaction>{
+    override fun success(response: OMGResponse<Transaction>) {
+        // Do something
+    }
+
+    override fun fail(response: OMGResponse<APIError>) {
+        // Handle error
+    }
+})
+```
+
+There are different ways to initialize a `TransactionCreateParams` by specifying either `toAddress`, `toAccountId` or `toProviderUserId`.
+
+### PaginationList
+`PaginationList` is an object represents a paginated filtered data set. 
+This object will be returned from any APIs that probably contain multiple objects (e.g. account list, transaction list, token list, etc.).
```

Comment:
`This object will be returned from any API listing resources (e.g. accounts, transactions, tokens, etc.).